client/src/document/ingredients/browser-compatibility-table/atoms/_bc-table.scss

```diff
@@ -55,10 +55,12 @@
   .bc-table-row-header {
     padding: math.div($base-spacing, 2) math.div($base-spacing, 4);
     display: flex;
+    display: -webkit-flex;
```

Comment:
That is so weird, I thought create-react-app takes care of this for us 🤔  It definitely does in other cases such as `gap`. Perhaps it is our support browser matrix set-up. Anyhow, thanks @tannerdolby

Reply:
Yeah I thought it was weird that it wasn't getting auto prefixed somewhere too. I think its probably the support matrix setup as well, probably worth taking a look at. I will try to see if I could track that down and provide a fix to ensure `display: flex` is being prefixed properly for all instances. 

Your welcome! @schalkneethling